.github/workflows/unit-test.yml

```diff
@@ -158,7 +158,6 @@ jobs:
             - name: Docker debug information
               run: |
                   docker -v
-                  docker-compose -v
```

Comment:
Is this temporary to check what's causing the failures?

Reply:
I think this is what was causing the failures actually. I guess these days it's `docker compose` and not `docker-compose` and Github probably updated docker or something so the PHP tests started failing, I expect that they'll fail elsewhere as well.

Reply:
Shall we merge this so we fix the tests in other PRs as well?